### PR TITLE
Lesson 6 part 1 (Decorator)

### PR DESCRIPTION
**Pull request by ViktoryiaKakoshanka**

### CODE REVIEW

File: SixthLesson/DecoratorStream/DecoratorStream/Decorators/PasswordRequestBeforeReading.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using System.IO;
+using DecoratorStream.Model;
+using DecoratorStream.View;
+
+namespace DecoratorStream.Decorators
+{
+    public class PasswordRequestBeforeReading : BaseDecoratorStream
```
Comment (AliaksandrMakarchuk) [inline SixthLesson/DecoratorStream/DecoratorStream/Decorators/PasswordRequestBeforeReading.cs]: Need to rename. At least, add **Decorator** at the end of the name. Or rename to e.g. **RequestPasswordDecorator**

File: SixthLesson/DecoratorStream/DecoratorStream/Decorators/TextLoader.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using DecoratorStream.View;
+
+namespace DecoratorStream.Decorators
+{
+    class TextLoader : BaseDecoratorStream
```
Comment (AliaksandrMakarchuk) [inline SixthLesson/DecoratorStream/DecoratorStream/Decorators/TextLoader.cs]: Need to rename.

File: SixthLesson/DecoratorStream/DecoratorStream/Decorators/TextLoader.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    {
+        private const int COUNTBYTESTOREAD = 30;
+
+        public TextLoader(Stream stream, IConsoleView view) : base(stream, view)     { }
+
+        public override int Read(byte[] buffer, int offset, int count)
```
Comment (AliaksandrMakarchuk) [inline SixthLesson/DecoratorStream/DecoratorStream/Decorators/TextLoader.cs]: Method is too big. I think you could extract loop body into separate method

File: SixthLesson/DecoratorStream/DecoratorStream/Decorators/PasswordRequestBeforeReading.cs (hunk 1 above)
Comment (ViktoryiaKakoshanka) [reply SixthLesson/DecoratorStream/DecoratorStream/Decorators/PasswordRequestBeforeReading.cs]: Renamed RequestPasswordDecorator

File: SixthLesson/DecoratorStream/DecoratorStream/Decorators/TextLoader.cs (hunk 2 above)
Comment (ViktoryiaKakoshanka) [reply SixthLesson/DecoratorStream/DecoratorStream/Decorators/TextLoader.cs]: Renamed **ProgressReadDecorator**

**Review by AliaksandrMakarchuk — APPROVED** [review]

**Pull request merged**